.github/workflows/lint-test.yaml

```
@@ -38,9 +43,9 @@ jobs:
         run: ct lint --config ct.yaml
 
       - name: Create k3d cluster
-        uses: nolar/setup-k3d-k3s@v1
+        uses: nolar/setup-k3d-k3s@293b8e5822a20bc0d5bcdd4826f1a665e72aba96 # v1.0.9
         with:
-          version: v1.25
+          version: v1.31
```

should fix the merge conflict
```suggestion
          version: v1.32
```